### DISCUSSION TITLE
Numerical instability - Wildfire simulation

### DISCUSSION THREAD

**Discussion by marialuiza-23** [firemodels/fds#12234] [Q&A]
I am trying to perform an analysis to simulate wildfire on FDS 6.8.0, but it is terminating due to numerical instabilities. What could I change to fix it?

![Screenshot 2023-11-16 140616](https://github.com/firemodels/fds/assets/144738283/11bc8b5d-d423-4302-b38e-f278725b11b7)

&HEAD CHID='SIM_EXT_V4' /

&MESH ID='Mesh', IJK=60,56,120, XB=0.0,15.0,0.0,14.0,0.0,30.0/

&TIME T_END=300. /
&DUMP DT_PL3D=10.0, DT_RESTART=300.0, DT_SL3D=0.25, WRITE_XYZ=.TRUE., PLOT3D_QUANTITY(1:2)='HRRPUV','TEMPERATURE'/

&MISC SIMULATION_MODE='LES', TMPA=28./

&WIND SPEED=6.0, DIRECTION=270., L=350., Z_0=0.03 /

&REAC FUEL='FUEL VAPOR', C=2.10, H=6.20, O=2.16, SOOT_YIELD=0.01, HEAT_OF_COMBUSTION=17425., IDEAL=T /
&SPEC ID='WATER VAPOR' /

&VENT MB='XMIN', SURF_ID='PERIODIC FLOW ONLY' /
&VENT MB='XMAX', SURF_ID='PERIODIC FLOW ONLY' /

&VENT MB='YMIN', SURF_ID='OPEN' /
&VENT MB='YMAX', SURF_ID='OPEN' /

&VENT MB='ZMAX', SURF_ID='OPEN' /

&MATL ID		= 'MINERAL_WOOL',
      SPECIFIC_HEAT	= 0.67,
      CONDUCTIVITY	= 0.042,
      DENSITY		= 100.0,
      EMISSIVITY	= 0.7/

&MATL ID 		= 'MOISTURE'
      DENSITY 		= 1000.
      CONDUCTIVITY 	= 0.6
      SPECIFIC_HEAT 	= 4.190
      REFERENCE_TEMPERATURE = 100.
      A 		= 600000.
      E 		= 48200.
      N_T 		= -0.5
      NU_SPEC 		= 1.
      SPEC_ID 		= 'WATER VAPOR'
      HEAT_OF_REACTION 	= 2259. /

&MATL ID                    = 'GENERIC VEGETATION'
      DENSITY               = 500.
      CONDUCTIVITY          = 0.2
      SPECIFIC_HEAT_RAMP    = 'c_v'
      A                     = 1040.
      E                     = 61041.
      NU_SPEC               = 0.75
      SPEC_ID               = 'FUEL VAPOR'
      NU_MATL               = 0.25
      MATL_ID               = 'CHAR'
      HEAT_OF_REACTION      = 418. /

&MATL ID                    = 'CHAR'
      DENSITY               = 300.
      CONDUCTIVITY          = 0.052
      SPECIFIC_HEAT_RAMP    = 'c_v'
      SURFACE_OXIDATION_MODEL = T
      A                  = 465.
      E                  = 68000. 
      SPEC_ID               = 'PRODUCTS','AIR'
      NU_SPEC               = 8.13,-7.17
      MATL_ID               = 'ASH'
      NU_MATL               = 0.04
      HEAT_OF_REACTION      = -25000. /

&MATL ID                    = 'ASH'
      DENSITY               = 67.
      CONDUCTIVITY          = 0.1
      SPECIFIC_HEAT_RAMP    = 'c_v' /

&RAMP ID='c_v', T=  0., F=1.1 /
&RAMP ID='c_v', T=200., F=2.0 /
&RAMP ID='c_v', T=800., F=2.0 /

&MATL ID		    = 'DIRT'
      CONDUCTIVITY 	    = 0.25
      SPECIFIC_HEAT 	    = 2.
      DENSITY 		    = 1300. /

&SURF ID			= 'GROUND'
      DEFAULT			= T
      MATL_ID			= 'DIRT'
      THICKNESS			= 0.1 /

&SURF ID			= 'Facade',
      FYI			= 'Painel 75 mm',
      RGB			= 146,202,166,
      BACKING			= 'VOID',
      MATL_ID(1,1)		= 'MINERAL_WOOL',
      MATL_MASS_FRACTION(1,1)	= 1.0,
      THICKNESS(1)		= 0.0375/

&SURF ID                        = 'foliage'
      MATL_ID(1,1) = 'GENERIC VEGETATION'
      MATL_MASS_FRACTION(1,1)   = 1.0
      MOISTURE_FRACTION         = 0.14
      SURFACE_VOLUME_RATIO      = 3940.
      LENGTH                    = 0.05
      GEOMETRY                  = 'CYLINDRICAL' /

&SURF ID                        = 'small roundwood'
      MATL_ID(1,1) = 'GENERIC VEGETATION'
      MATL_MASS_FRACTION(1,1)   = 1.0
      MOISTURE_FRACTION         = 0.14
      SURFACE_VOLUME_RATIO      = 2667.
      LENGTH                    = 0.10
      GEOMETRY                  = 'CYLINDRICAL' /

&SURF ID                        = 'medium roundwood'
      MATL_ID(1,1) = 'GENERIC VEGETATION'
      MATL_MASS_FRACTION(1,1)   = 1.0
      MOISTURE_FRACTION         = 0.14
      SURFACE_VOLUME_RATIO      = 888.
      LENGTH                    = 0.10
      GEOMETRY                  = 'CYLINDRICAL' /

&SURF ID                        = 'large roundwood'
      MATL_ID(1,1) = 'GENERIC VEGETATION'
      MATL_MASS_FRACTION(1,1)   = 1.0
      MOISTURE_FRACTION         = 0.14
      SURFACE_VOLUME_RATIO      = 500.
      LENGTH                    = 0.10
      GEOMETRY                  = 'CYLINDRICAL' /

&SURF ID                        = 'wet vegetation'
      MATL_ID(1,1:1)            = 'GENERIC VEGETATION'
      MOISTURE_FRACTION         = 0.058
      SURFACE_VOLUME_RATIO      = 12240.
      LENGTH                    = 0.20
      GEOMETRY                  = 'CYLINDRICAL' /

&OBST ID='Obst_Ignition', XB=0.0,0.75,0.0,14.0,0.0,0.01, THICKEN=.TRUE., SURF_ID='IGNITION'/ 

&SURF ID='IGNITION', HRRPUA=850., COLOR='RED', TMP_FRONT=100., RAMP_Q='ignitor' /

&RAMP ID='ignitor', T= 0., F=0. /
&RAMP ID='ignitor', T= 1., F=1. /
&RAMP ID='ignitor', T=11., F=1. /
&RAMP ID='ignitor', T=12., F=0. /

&PART ID='grass', SAMPLING_FACTOR=1, SURF_ID='wet vegetation', PROP_ID='grass image', DRAG_COEFFICIENT=2.8
      QUANTITIES='PARTICLE TEMPERATURE','PARTICLE MASS', STATIC=T, COLOR='BROWN' /

&PART ID='foliage', DRAG_COEFFICIENT=2.8, SAMPLING_FACTOR=1, SURF_ID='foliage'
      QUANTITIES='PARTICLE TEMPERATURE','PARTICLE MASS', STATIC=.TRUE., COLOR='FOREST GREEN' /

&PART ID='small roundwood', DRAG_COEFFICIENT=2.8, SAMPLING_FACTOR=1, SURF_ID='small roundwood'
      QUANTITIES='PARTICLE TEMPERATURE','PARTICLE MASS','PARTICLE DIAMETER', STATIC=.TRUE., COLOR='BROWN' /

&PART ID='medium roundwood', DRAG_COEFFICIENT=2.8, SAMPLING_FACTOR=1, SURF_ID='medium roundwood'
      QUANTITIES='PARTICLE TEMPERATURE','PARTICLE MASS','PARTICLE DIAMETER', STATIC=.TRUE., COLOR='BROWN' /

&PART ID='large roundwood', DRAG_COEFFICIENT=2.8, SAMPLING_FACTOR=1, SURF_ID='large roundwood'
      QUANTITIES='PARTICLE TEMPERATURE','PARTICLE MASS','PARTICLE DIAMETER', STATIC=.TRUE., COLOR='BROWN' /

&INIT ID='grass', PART_ID='grass', N_PARTICLES_PER_CELL=1, CELL_CENTERED=.TRUE., MASS_PER_VOLUME=0.46144, XB=0.0,15.0,0.0,14.0,0.0,0.30, DRY=T/ 

# --------- ROW 01 ---------

&INIT PART_ID='foliage', XYZ=4.0,1.95,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=1.2336 /
&INIT PART_ID='small roundwood', XYZ=4.0,1.95,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.3495 /
&INIT PART_ID='medium roundwood', XYZ=4.0,1.95,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.2467 /
&INIT PART_ID='large roundwood', XYZ=4.0,1.95,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.2262 /

&INIT PART_ID='foliage', XYZ=4.0,7.0,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=1.2336 /
&INIT PART_ID='small roundwood', XYZ=4.0,7.0,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.3495 /
&INIT PART_ID='medium roundwood', XYZ=4.0,7.0,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.2467 /
&INIT PART_ID='large roundwood', XYZ=4.0,7.0,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.2262 /

&INIT PART_ID='foliage', XYZ=4.0,12.05,0.30, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=1.2336 /
&INIT PART_ID='small roundwood', XYZ=4.0,12.05,0.30, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.3495 /
&INIT PART_ID='medium roundwood', XYZ=4.0,12.05,0.30, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.2467 /
&INIT PART_ID='large roundwood', XYZ=4.0,12.05,0.30, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.2262 /

# --------- ROW 02 ---------

&INIT PART_ID='foliage', XYZ=8.0,1.95,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=1.2336 /
&INIT PART_ID='small roundwood', XYZ=8.0,1.95,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.3495 /
&INIT PART_ID='medium roundwood', XYZ=8.0,1.95,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.2467 /
&INIT PART_ID='large roundwood', XYZ=8.0,1.95,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.2262 /

&INIT PART_ID='foliage', XYZ=8.0,7.0,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=1.2336 /
&INIT PART_ID='small roundwood', XYZ=8.0,7.0,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.3495 /
&INIT PART_ID='medium roundwood', XYZ=8.0,7.0,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.2467 /
&INIT PART_ID='large roundwood', XYZ=8.0,7.0,0.0, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.2262 /

&INIT PART_ID='foliage', XYZ=8.0,12.05,0.30, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=1.2336 /
&INIT PART_ID='small roundwood', XYZ=8.0,12.05,0.30, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.3495 /
&INIT PART_ID='medium roundwood', XYZ=8.0,12.05,0.30, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.2467 /
&INIT PART_ID='large roundwood', XYZ=8.0,12.05,0.30, RADIUS=1.45, HEIGHT=4.3, SHAPE='CONE', N_PARTICLES_PER_CELL=1, DRY=T, MASS_PER_VOLUME=0.2262 /

&OBST ID='PAR 1', XB=12.0,12.0,1.0,13.0,0.0,6.0, SURF_ID='Facade'/ 

cDEVC ID='TP_PAR01_01_2m', QUANTITY='THERMOCOUPLE', XYZ=34.0,14.0,2.0, ORIENTATION=-1.0,0.0,0.0/
cDEVC ID='TP_PAR01_01_4,5m', QUANTITY='THERMOCOUPLE', XYZ=34.0,14.0,4.5, ORIENTATION=-1.0,0.0,0.0/
cDEVC ID='TP_PAR01_01_INT_2m', QUANTITY='THERMOCOUPLE', XYZ=34.0,14.0,2.0, ORIENTATION=-1.0,0.0,0.0/

&DEVC ID='AST_01', XYZ=12.0,5.0,2.0, QUANTITY='ADIABATIC SURFACE TEMPERATURE GAS', ORIENTATION=-1.0,0.0,0.0, PROP_ID='props' /
&DEVC ID='AST_02', XYZ=12.0,9.0,2.0, QUANTITY='ADIABATIC SURFACE TEMPERATURE GAS', ORIENTATION=-1.0,0.0,0.0, PROP_ID='props' /
&DEVC ID='AST_03', XYZ=12.0,5.0,4.5, QUANTITY='ADIABATIC SURFACE TEMPERATURE GAS', ORIENTATION=-1.0,0.0,0.0, PROP_ID='props' /
&DEVC ID='AST_04', XYZ=12.0,9.0,4.5, QUANTITY='ADIABATIC SURFACE TEMPERATURE GAS', ORIENTATION=-1.0,0.0,0.0, PROP_ID='props' /

&PROP ID='props', EMISSIVITY=0.9, HEAT_TRANSFER_COEFFICIENT=10. /

&SLCF PBX=20., QUANTITY='TEMPERATURE', VECTOR=T /
&SLCF PBX=33., QUANTITY='TEMPERATURE', VECTOR=T /
&SLCF PBY=7., QUANTITY='TEMPERATURE', VECTOR=T /

&TAIL /

**Comment by mcgratta** [comment]
This Discussion has been converted to an Issue.
